app/assets/javascripts/application.js.erb

```diff
@@ -727,4 +727,19 @@ onPageLoad(() => {
         response.text().then(text => replace(comment, text));
       });
   });
+
+  on('click', '.comment_unread', (event) => {
+    var nodes = document.getElementsByClassName('comment_unread')
+    var foundIndex = Array.from(nodes).findIndex(function(n){
+      if (n.id == event.target.id) {
+        return true;
+      }
+    })
+
+    var targetIndex = foundIndex + 1
```

Comment:
`targetIndex = (foundIndex + 1) % nodes.length;`

Then this wraps around, a nice feature for people who start from the middle.

It also means you can remove the `if` from the next line. (I thought about making it `foundIndex != targetIndex` so that we don’t try to scroll to the current comment when there’s only one unread comment, but if nothing happens it just looks broken - recentering the comment at least acknowledges the click.)